students/Kristy_Martini/lesson01/break_me.py

```diff
@@ -0,0 +1,41 @@
+def createNameError():
```

Comment:
Functions (and almost everything except classes) should be snake_case in Python: create_name_error():